tests/snmp/test_snmp_phy_entity.py

```diff
@@ -99,6 +99,14 @@
 XCVR_DOM_KEY_TEMPLATE = 'TRANSCEIVER_DOM_SENSOR|{}'
 
 
+@pytest.fixture(autouse=True, scope="module")
+def check_image_version(duthost):
+    """Skip the test for unsupported images."""
+    if "201911" in duthost.os_version:
+        pytest.skip('Test not supported for 201911 images. Skipping the test')
```

Comment:
why not use "pytest_require"?

Reply:
Sure, that is better. Updated now. Thanks!

Reply:
In a separate PR, I can also replace other `pytest.skip` in this file with `pytest_require`.